README.md

```diff
@@ -24,17 +24,26 @@ Procedure:
     
     4b.To run the code in local, open an Ubuntu terminal and follow the below steps:  
         4b-1: Clone into the github branch (SUDSM_HARTFORD_ASSIGNMENT1)  
-        4b-2: Open and activate a virtual environment (name:htassign)  
-        4b-3: Instal the requirements file  
+        4b-2: Open and activate a virtual environment (name:htassign) 
+                For creating virtual environment, run "python3 -m venv htassign"  
+                Now to activate it run source "htassign/bin/activate"  
+                After running this the virtual environment would be activated.  
+        4b-3: Install the requirements file  
         4b-4: Install awscli and configure using the s3 access ID and Key  
+                Install awscli by typing in "pip install awscli" in the terminal  
+                Check the version of awscli by using "aws --version" (I have installed version "aws-cli/1.19.53")  
+                Post installation of awscli, to get access to the S3 bucket type "aws configure"  
+                This would ask for the access ID, access key, region and output format  
+                Enter the details as downloaded from aws while creating S3 bucket  
         4b-5: Run the application using python app.py  
-  
-        ![image](https://user-images.githubusercontent.com/82586609/115119361-93160680-9fc5-11eb-99ab-f3925441f530.png)  
-  
-        ![image](https://user-images.githubusercontent.com/82586609/115119367-a0cb8c00-9fc5-11eb-8050-5dcf67bd17d6.png)  
-  
-        ![image](https://user-images.githubusercontent.com/82586609/115119439-fa33bb00-9fc5-11eb-93c4-b27057a909c8.png)  
-  
+                Before running, check if python is installed by running "python --version"  
+                If not installed, try the below commands in order:  
+                sudo apt update  
```

Comment:
Whenever you are putting in code, please use like this
```bash
sudo apt update
```
This is a better way to mention steps.

Reply:
Got it!